lib/linguist/heuristics.rb

```diff
@@ -380,6 +380,16 @@ def call(data)
       end
     end
 
+    disambiguate ".pod" do |data|
+      if /^\s*\/\* XPM \*\//.match(data)
```

Comment:
This was copied from `.pm`, but I think isn't necessary for pod.

Reply:
Have removed from `heuristics.yml` on the tip of this branch. Haven't regenerated a gem for testing because it won't actually come up.